maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -423,33 +422,25 @@ class IOSDriver(
     override fun backPress() {}
 
     override fun hideKeyboard() {
-        val future = executor.submit {
```

Comment:
I'm curious, do you happen to know why it used a different thread to run the hideKeyboard?